easybuild/easyconfigs/a/ancestry/ancestry-1.0.0-GCCcore-8.3.0.eb

```diff
@@ -0,0 +1,35 @@
+# easybuild easyconfig
+#
+# John Dey <jfdey@fredhutch.org>
+#
+# Fred Hutchinson Cancer Research Center - Seattle Washington - US
+#
+easyblock = 'MakeCp'
+
+name = 'ancestry'
+version = '1.0.0'
+
+homepage = 'https://github.com/vibansal'
+description = """Fast individual ancestry inference from DNA sequence data leveraging allele frequencies
+ from multiple populations. iAdmix Using population allele frequencies for computing individual admixture
+ estimates"""
+
+toolchain = {'name': 'GCCcore', 'version': '8.3.0'}
+
+dependencies = [
+    ('binutils', '2.32'),
+    ('ncurses', '6.1'),
+]
+
+source_urls = ['https://github.com/vibansal/ancestry/archive']
+sources = ['%(version)s.tar.gz']
+checksums = ['1e8d3ff5fb422d2723d04850959868362d83bc0d3db47b1dd565cc84c255463a']
+
+files_to_copy = [(['calculateGLL', 'ANCESTRY'], 'bin'), (['lbfgsb.o'], 'lib')]
```

Comment:
why copy the object file? I see the author also checked in a compiled object file into his git repo (big ouch) as well.